src/admin/tests/admin.e2e.ts

```diff
@@ -0,0 +1,103 @@
+import { test, expect } from '@playwright/test'
+import { HttpStatus } from '@nestjs/common'
+import {
+  loginUser,
+  registerUser,
+  deleteUser,
+  generateRandomEmail,
+  generateRandomName,
+} from '../../../e2e-tests/utils/auth.util'
+
+test.describe('Admin - Campaign Management', () => {
+  const adminEmail = process.env.ADMIN_EMAIL
+  const adminPassword = process.env.ADMIN_PASSWORD
+
+  test.beforeAll(() => {
+    test.skip(!adminEmail || !adminPassword, 'Admin credentials not configured')
+  })
+
+  let adminToken: string
+  let testUserId: number | undefined
+  let testAuthToken: string | undefined
+
+  test.afterEach(async ({ request }) => {
+    if (testUserId && testAuthToken) {
+      await deleteUser(request, testUserId, testAuthToken)
+      testUserId = undefined
+      testAuthToken = undefined
+    }
```

Comment:
The `afterEach`es in this PR aren't using the parallel-safe logic that the last one did.